src/forms/CalendarComponent.jsx

```diff
@@ -145,31 +143,27 @@ class CalendarComponent extends React.Component {
 
 
 		return (
-			<div>
-				<span>
-					{label && (
-						<label htmlFor={id} className={classNames.label}>
-							{label}
-						</label>
-					)}
-					{helperText &&
-						<div className={classNames.helperText}>
-							{helperText}
-						</div>
-					}
-					<input
-						type="text"
-						id={id}
-						name={name}
-						defaultValue={value}
-						className={classNames.field}
-						ref={input => (this.inputEl = input)}
-						{...other}
-					/>
-				</span>
-				{!suppressError &&
-					error && <p className="text--error text--small">{error}</p>}
-			</div>
+			<span>
+				{label && (
+					<label htmlFor={name} className={classNames.label}>
+						{label}
+					</label>
+				)}
+				{helperText &&
+					<div className={classNames.helperText}>
+						{helperText}
+					</div>
+				}
+				<input
+					type="text"
+					id={name}
+					name={name}
+					defaultValue={value}
+					className={classNames.field}
+					ref={input => (this.inputEl = input)}
```

Comment:
I think there's an AirBnB eslint rule that disallows variable assignment inside return values - might want to clean that up now

Reply:
added a file-level disable for that rule

Reply:
I recommend fixing the code rather than disabling the rule - it's useful outside of this line.